operator/v1/types_network.go

```diff
@@ -433,6 +447,18 @@ type OVNKubernetesConfig struct {
 	// fields within ipv4 for details of default values.
 	// +optional
 	IPv6 *IPv6OVNKubernetesConfig `json:"ipv6,omitempty"`
+
+	// routeAdvertisements determines if the functionality to advertise cluster
+	// network routes through BGP is enabled or not. This functionality is
```

Comment:
maybe just say advertise cluster routes through a dynamic routing protocol, such as BGP? In the future we might extend routeAdvertisements to leverage other routing protocols as well.